src/main/java/ome/files/performance/TilingPerformance.java

```diff
@@ -0,0 +1,336 @@
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package ome.files.performance;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import loci.formats.ome.OMEXMLMetadata;
+import loci.formats.ome.OMEXMLMetadataImpl;
+import loci.formats.out.OMETiffWriter;
+import loci.formats.out.TiffWriter;
+import loci.formats.tiff.IFD;
+import ome.xml.model.primitives.PositiveInteger;
+import loci.formats.FormatWriter;
+import loci.formats.ImageReader;
+
+public final class TilingPerformance {
+
+  public static void main(String[] args) {
+    if (args.length != 12) {
+      System.out.println("Usage: TilingPerformance iterations tileSizeXStart tileSizeYStart tileSizeXEnd tileSizeYEnd tileSizeOperator tileSizeIncrement series autoTile inputfile outputfile resultfile\n");
+      System.exit(1);
+    }
+
+    Result result = null;
+
+    try {
+      int iterations = Integer.parseInt(args[0]);
+      int tileSizeXStart = Integer.parseInt(args[1]);
+      int tileSizeYStart = Integer.parseInt(args[2]);
+      int tileSizeXEnd = Integer.parseInt(args[3]);
+      int tileSizeYEnd = Integer.parseInt(args[4]);
+      String tileSizeOperator = args[5];
+      int tileSizeIncrement = Integer.parseInt(args[6]);
+      int series = Integer.parseInt(args[7]);
+      boolean autoTiling = Boolean.parseBoolean(args[8]);
+      Path infile = Paths.get(args[9]);
+      String outfileBase = args[10];
+      Path resultfile = Paths.get(args[11]);
+
+      
+      result = new Result(resultfile);
+
+      for(int i = 0; i < iterations; i++) {
+        int tileSizeX = tileSizeXStart;
+        int tileSizeY = tileSizeYStart;
+        while (tileSizeX >= tileSizeXEnd) {
+          tileSizeY = tileSizeYStart;
+          while (tileSizeY >= tileSizeYEnd) {
+            System.out.println("New tileX " + tileSizeX + " " + tileSizeY + " " + autoTiling);
+            result.addCustomParam("test.tileSizeX", tileSizeX);
+            result.addCustomParam("test.tileSizeY", tileSizeY);
+            Path outfile = Paths.get(outfileBase+"-"+tileSizeX+"-"+tileSizeY+".ome.tiff");
+            System.out.print("pass " + i + ": init...");
+            System.out.flush();
+            OMEXMLMetadata meta = new OMEXMLMetadataImpl();
+            List<List<byte[]>> pixels = new ArrayList<List<byte[]>>();
```

Comment:
Would this be better named `planes`, since it's a list of planes containing a list of tiles?